src/CommandBus.php

```diff
@@ -74,12 +74,12 @@ public function dispatch($command)
                     $this->processCommand($command);
                 }
                 $this->isDispatching = false;
-            } catch (\Throwable $ex) {
-                $this->isDispatching = false;
-                throw $ex;
             } catch (\Exception $e) {
                 $this->isDispatching = false;
                 throw CommandDispatchException::wrap($e, $this->commandQueue);
+            } catch (\Throwable $e) {
+                $this->isDispatching = false;
+                throw $e;
```

Comment:
why not wrap this one in CommandDispatchException?

Reply:
because of the type hint in CommandDispatchException

Reply:
alright